.gitignore

```diff
@@ -48,8 +48,6 @@ graph.dot
 
 prisma-schema-wasm/nodejs
 
-# Ignore pnpm-lock.yaml
-query-engine/driver-adapters/pnpm-lock.yaml
```

Comment:
thanks

Reply:
actually I don't see it committed in this PR, is also covered by a `.gitignore` in some subdirectory?